android/android-BluetoothLeGatt.iml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
There are two IML files. Do we need them? I guess importing the gradle file works fine as well.

Reply:
done